compiler/noirc_frontend/src/elaborator/mod.rs

```diff
@@ -1171,10 +1171,10 @@ impl<'context> Elaborator<'context> {
             let attributes = std::mem::take(&mut typ.struct_def.attributes);
             let span = typ.struct_def.span;
 
-            let (fields, generics) = self.resolve_struct_fields(typ.struct_def, type_id);
+            let fields = self.resolve_struct_fields(typ.struct_def, type_id);
             self.interner.update_struct(type_id, |struct_def| {
                 struct_def.set_fields(fields);
-                struct_def.generics = generics;
+                // struct_def.generics = generics;
```

Comment:
Commented code. Is this no longer needed?

Reply:
yes removed